packages/calcite-components/src/components/rating/rating.scss

```diff
@@ -15,17 +20,17 @@
 
 :host([scale="s"]) {
   @apply h-6;
-  --calcite-rating-spacing-unit: theme("spacing.1");
+  --spacing: theme("spacing.1");
```

Comment:
tokens should not be assigned in the component unless they are tagged with `-internal-`